tool/templates/cpp/runtime.hpp

```diff
@@ -1,26 +1,31 @@
 #ifndef DIPLOMAT_RUNTIME_CPP_H
 #define DIPLOMAT_RUNTIME_CPP_H
 
-#include <string>
-#include <variant>
-#include <array>
 #include <optional>
+#include <string>
 #include <type_traits>
+#include <variant>
 
 #if __cplusplus >= 202002L
-#include<span>
+#include <span>
+#else
+#include <array>
 #endif
 
-#include "diplomat_c_runtime.hpp"
-
 namespace diplomat {
 
-extern "C" inline void Flush(capi::DiplomatWrite* w) {
+namespace capi {
+extern "C" {
+// SHARED_CAPI
+} // extern "C"
+} // namespace capi
+
+extern "C" inline void _Flush(capi::DiplomatWrite* w) {
```

Comment:
issue: no, this isn't allowed

https://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf

> All identifiers that begin with an underscore and either an uppercase letter or another underscore are always reserved for any use.

Reply:
> All identifiers that begin with an underscore are always reserved for use as identifiers with file scope in both the ordinary and tag name spaces.

That's exactly what I want. So underscore lowercase?

Reply:
Yes.